### DISCUSSION TITLE
What of data sources should our curb Events specification support?

### DISCUSSION THREAD

**Discussion by jfh01** [openmobilityfoundation/curb-data-specification#8] [Ideas]
We have discussed the need for the events specification to accommodate data from a variety of sources. Some of the ones that have been mentioned are:

- In-ground / meter-mounted sensors
- Computer vision / fixed cameras
- ALPRs (fixed or mobile)
- Fleet-reported data
- Meter data (e.g. parking sessions)
- Bluetooth / RFID (need to define in more detail)

What other kinds of data sources should we accommodate? Please also share specific companies and technologies you are familiar with so that we can research/reach out as part of the design process.

**Comment by jiffyclub** [comment]
Maybe GPS coordinates/duration or GPS trip routes and timing?

**Comment by ITE-Curbside** [comment]
Street centerline data in the SharedStreets linear reference format.

**Comment by ITE-Curbside** [comment]
CurbLR data standard for describing curb regulations.

**Comment by ITE-Curbside** [comment]
CurbIQ: https://curbiq.io/

**Comment by ITE-Curbside** [comment]
Mapillary physical feature data.

**Comment by ITE-Curbside** [comment]
Coord app for curb uses, TOD, and revenue data as well as Coord Collector to geocode physical features.

**Comment by ITE-Curbside** [comment]
INRIX Road Rules platform.

**Comment by jacobmalleau** [comment]
You mentioned meter data for payment sessions, but we should also accommodate payment session data from parking payment apps.

**Comment by schnuerle** [comment]
Many of the geographic/road suggestions here seem to apply to the Curbs API, not Events API.

For payment session data in Events, `actual_cost` and `currency` has been added to the event object as optional, aligning with MDS:

https://github.com/openmobilityfoundation/curb-data-specification/blob/feature-release-work-1/events/README.md#curb-event
